js/front/dataset/index.js

```diff
@@ -72,21 +71,15 @@ new Vue({
         },
 
         /**
-         * Extract the current dataset metadatas from microdata markup
+         * Extract the current dataset metadatas from JSON-LD script
          * @return {Object} The parsed dataset
          */
         extractDataset() {
-            const dataset = microdata('http://schema.org/Dataset')[0];
-            if (Array.isArray(dataset.distribution)) {
-                dataset.resources = dataset.distribution;
-            } else {
-                dataset.resources = dataset.distribution ? [dataset.distribution] : [];
-            }
+            const selector = "script[type='application/ld+json']";
```

Comment:
I put the ID.

For the move to a separate file, I am bit more dubious. Testing it is not very useful (actually, we would basically test JSON.parse and querySelector) and so far, it is used for dataset only.